DataFormats/MuonReco/src/Muon.cc

```diff
@@ -68,6 +82,12 @@ int Muon::numberOfMatches( ArbitrationType type ) const
          matches += chamberMatch->rpcMatches.size();
          continue;
       }
+      
+      if(type == ME0HitAndTrackArbitration) {
+         if(chamberMatch->me0Matches.empty()) continue;
+         matches += chamberMatch->me0Matches.size();
```

Comment:
Line 87 is unnecessary. If the vector is `empty()`, then `size()` will equal zero. Line 87 can be deleted, and, in the case of an empty vector, zero will be added to `matches`, to no effect.